### DISCUSSION TITLE
Static image w/videos and Library shows 'empty' with 40k photos

### DISCUSSION THREAD

**Discussion by canosgmk** [photoprism/photoprism#3661] [Q&A]
Hello, 

Photoprism 16.0.0 on TrueNAS Scale 22.12.3.3 was fully operational and working with no complaits from my family.  We where even watching videos and going through photos on our iphones, ipads, PC Laptops for the last two months (the family enough having a photo server to play with).  However, NOW, with no changes to the server, OS version or App, Photoprism is showing an empty Library/Originals with  40K+ photos in it.  

The search section still has all the photos.  The calendar still has all the photos.  However ALL the videos (900+) are now "Missing".  Which is odd cause all the data is still listed in size in the dataset.  

ideas what happen?  

Again, this was fully functional a week or two ago.  we didn't change anything to TrueNAS or Photoprism.

**Comment by canosgmk** [comment]
![Photoprism search](https://github.com/photoprism/photoprism/assets/95102447/127647b5-1ae8-4058-be15-aa3f2152a377)
![Photoprism datsset](https://github.com/photoprism/photoprism/assets/95102447/9c852652-6433-4caa-91fe-47643d0b8d2e)
![Phtooprism error](https://github.com/photoprism/photoprism/assets/95102447/2253caaf-4adb-47aa-8837-91662667a5ca)
![Photoprism video](https://github.com/photoprism/photoprism/assets/95102447/4a48a06f-49fe-471f-9a3d-59168b897984)

**Comment by canosgmk** [comment]
![image](https://github.com/photoprism/photoprism/assets/95102447/7ea4416d-198c-4068-9322-3da87131d2eb)
update.  everyday, photoprism is deleting something as none of the files now can be found, however the data is in the dataset and the ram count is still shown on the website.

**Comment by canosgmk** [comment]
Would love some help on this.  

I have tried to contact almost all photoprism platform.  This is a massive glitch and need some guidance where i should look to fix it.  Clearly, cause it was operational after loading all the videos and photos for a few weeks, the program did something and lost all the addresses of the photos

**Reply by lastzero** [reply]
Could be some configuration or filesystem issue. PhotoPrism isn't deleting anything on its own.
